pkg/ext-proc/handlers/request.go

```diff
@@ -50,12 +50,12 @@ func (s *Server) HandleRequestBody(
 	// are able to be requested by using their distinct name.
 	modelObj := s.datastore.FetchModelData(model)
 	if modelObj == nil {
-		return nil, fmt.Errorf("error finding a model object in InferenceModel for input %v", model)
+		return nil, errutil.Error{Code: errutil.Internal, Msg: fmt.Sprintf("error finding a model object in InferenceModel for input %v", model)}
```

Comment:
This shouldn't be `Internal`. This is due to misconfiguration from the user. Perhaps adding a new code `BadConfiguration` or something alike, or simply `ModelNotFound`.

Reply:
Changed it to `BadConfiguration`.